### PR TITLE
Implement executing the last run using --last

### PR DESCRIPTION
**Pull request by psss**
Covers story `/stories/cli/run/last` introduced in #136. I have renamed the option to `--last` as it sounds somehow more natural to me. @lukaszachy, @thrix, what do you think?

### CODE REVIEW

File: tmt/utils.py [hunk 1]
```diff
@@ -38,6 +38,41 @@
...
+                pass
+            try:
+                os.symlink(run_id, symlink)
+            except OSError as error:
+                raise GeneralError(
+                    f"Unable to save last run '{self.path}'.\n{error}")
+            return run_id
+        if os.path.islink(symlink):
+            return os.path.realpath(symlink)
+        return None
```
Comment (thrix) [inline tmt/utils.py:74]: I would rather blow up in this case. User asks to run last run, but instead a new run is launched?

Not sure if that is ok ...

```
$ rm -f ~/.config/tmt/last-run
$ tmt run --last
/var/tmp/tmt/run-121
/plans/basic
...
```

Comment (psss) [reply tmt/utils.py:74]: Good point! Should now be fixed:

    > rm  ~/.config/tmt/last-run
    > tmt run -l
    [ ERROR ] No last run id found. Have you executed any run?

**Comment by thrix** [comment]
@psss some modules will need some love to work properly ....

Run in tmt repo ...
```
$ tmt run -a provision -h container
$ tmt run --last 
/var/tmp/tmt/run-121

/plans/basic
    discover
        status: done
        summary: 6 tests selected
    provision
        status: done
        summary: 1 guest provisioned
    prepare
        status: done
        summary: 1 preparation applied
    execute
        how: beakerlib
    finish

stderr (1/1 lines)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Error: no container with name or ID tmt-var-tmp-tmt-run-121-plans-basic found: no such container

 ERROR  Failed to run command 'podman container stop tmt-var-tmp-tmt-run-121-plans-basic'.
```

Do we want to fix it here?

**Comment by psss** [comment]
Well, I would say this is a not supported scenario. You should either skip `finish` when running for the first time or use `tmt run -lf` to force creating a new container.

**Comment by thrix** [comment]
> Well, I would say this is a not supported scenario. You should either skip `finish` when running for the first time or use `tmt run -lf` to force creating a new container.

Yeah, this is not user friendly, I would just expect it to go through with some nice warning nothing to cleanup .... maybe

**Comment by thrix** [comment]
> > Well, I would say this is a not supported scenario. You should either skip `finish` when running for the first time or use `tmt run -lf` to force creating a new container.
> 
> Yeah, this is not user friendly, I believe the default scenario for rerunning should be easy like this ...

cleared up on IRC

**Comment by psss** [comment]
Thanks for the quick review!

**Pull request merged**